### DISCUSSION TITLE
Can host driver connection be detected in my control-surface code?

### DISCUSSION THREAD

**Discussion by Guod3** [tttapa/Control-Surface#1009] [Q&A]
I would like to have an LED light when the PC host application connects to the device driver input port of my control surface. I'm hoping there is some low level connection status that can be exposed to my code somehow...

**Comment by tttapa** [comment]
Which board are you using?

Unfortunately, there's no standard way to do this. See https://github.com/tttapa/Control-Surface/discussions/924#discussioncomment-6020816 for a bit more details.

**Comment by Guod3** [comment]
I'm using Teensy 2.0
Thinking about this more, the PC application for my control surface will be a custom developed interface, which can open the port for output and send a midi event to turn on a LED, and every now and then to keep it on.
If there was native way of achieving the connection status that would be better, of course. 
Thanks for the library!
